incubator/rethinkdb-cluster/templates/rethinkdb-service.yaml

```diff
@@ -0,0 +1,44 @@
+# A headless service to create DNS records
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ template "fullname" . }}-cluster"
+  labels:
+    app: "{{ template "name" . }}-cluster"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+  {{- if .Values.cluster.serviceAnnotations }}
+{{ toYaml .Values.cluster.serviceAnnotations | indent 4 }}
+  {{- end }}
+spec:
+  ports:
+    - port: {{ .Values.ports.cluster }}
+      targetPort: cluster
+  selector:
+    app: "{{ template "name" . }}-cluster"
+    release: {{ .Release.Name | quote }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}-proxy
+  labels:
+    app: {{ template "name" . }}-proxy
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  annotations:
+  {{- if .Values.proxy.serviceAnnotations }}
+{{ toYaml .Values.proxy.serviceAnnotations | indent 4 }}
+  {{- end }}
+spec:
+  type: LoadBalancer
```

Comment:
Please make service type etc. configurable. Here's a good example of how this can be done.
https://github.com/kubernetes/charts/blob/master/stable/nginx-ingress/templates/controller-service.yaml#L16